shared/schemas/quilt_summarize.json

```diff
@@ -63,6 +63,10 @@
             }
           }
         },
+        "config": {
+          "description": "Perspctive specific option. Restores config from .save()",
```

Comment:
@akarve pls wordsmith

Reply:
@akarve I'm going to merge it, feel free to point me out to different wording here on in Slack